### PR TITLE
Sending OPTIONS request to API

### PR DESCRIPTION
**Pull request by patgrudniewski**

### CODE REVIEW

File: tests/reducer.test.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import { expect } from 'chai';
+
+import reducer from '../src/reducer.js';
+import {
+	OPTIONS_LOADED_TYPE,
+	OPTIONS_LOAD_FAILED_TYPE,
+} from '../src/actions.js';
+
+const actionTypeDependentResultTest = (type, payload, prevState, expectedState) => () => {
+	const action = { type, payload };
```
Comment (patgrudniewski) [inline tests/reducer.test.js]: @Druzzdil fix indentation here

File: src/Loader.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+const mapStateToProps = state => ({
+  options: state.crud,
+});
+
+export default connect(mapStateToProps, dispatch => ({ dispatch }))(Loader);
+
+export {
+    Loader,
```
Comment (patgrudniewski) [inline src/Loader.js]: @Druzzdil indent

File: docs/api/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# API
+
+## [`reducer`](reducer.md)
+
+> The crud reducer. Have to be mounted to Redux store at `crud`.
```
Comment (auamarto) [inline docs/api/README.md]: should be at `crud` what ?

File: src/reducer.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      return {
+        ...state,
+        [payload.namespace]: payload.response,
+      };
+    default: 
+      return state;
+  }
+};
+
+export default reducer;
```
Comment (auamarto) [inline src/reducer.js]: new line

File: tests/Loader.test.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import { render, shallow} from 'enzyme'
+import React from 'react'
+import {Loader} from '../src/Loader.js';
+import Enzyme, { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+Enzyme.configure({ adapter: new Adapter() });
```
Comment (auamarto) [inline tests/Loader.test.js]: separate imports ?

**Review by auamarto — CHANGES_REQUESTED** [review]

**Review by auamarto — APPROVED** [review]

**Pull request merged**